test/libponyc/badpony.cc

```diff
@@ -1032,21 +1032,21 @@ TEST_F(BadPonyTest, ThisViewpointWithIsoReceiver)
   TEST_ERRORS_1(src, "argument not a subtype of parameter");
 }
 
-TEST_F(BadPonyTest, DisallowPointerAndMaybePointerInEmbeededType)
+TEST_F(BadPonyTest, DisallowPointerAndNullablePointerInEmbeededType)
 {
   // From issue #2596
-  const char* src =
+  const char *src =
```

Comment:
this shouldnt have the whitespace change here

Reply:
I see my editor is messing with formating. I have to tame it.